office365_admin/help.md

```diff
@@ -201,42 +250,44 @@ This action updates usage location for a given user.
 Example output:
 
 ```
-
 {
   "success": true
 }
-
 ```
 
-#### Get Subscribed SKUs
+### Triggers
 
-This action gets a list of commercial subscriptions that an organization has acquired.
+_This plugin does not contain any triggers._
 
-#### Output
+### Custom Output Types
+
+#### serviceItem
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|sku_item|[]skuItem|True|Information about a given SKU|
-
-Example output:
+|Applies To|string|True|Entity SKU applies to|
+|Service Plan ID|string|True|Service plan ID|
+|Service Plan Name|string|True|Service plan name|
 
-```
-```
+#### skuItem
 
-### Triggers
-
-This plugin does not contain any triggers.
-
-### Custom Output Types
-
-_This plugin does not contain any custom output types._
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|appliesTo|string|True|Entity SKU applies to|
+|Capability Status|string|True|Availability of SKU|
+|Consumed Units|integer|False|Consumed units|
+|ID|string|False|SKU team ID|
+|Service Plans|[]serviceItem|True|List of service plans|
+|SKU ID|string|True|SKU ID|
+|SKU Part Number|string|True|SKU part number|
 
 ## Troubleshooting
 
-This plugin does not contain any troubleshooting information.
+_This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.3.0 - Added action Lookup User By Email
```

Comment:
Did you add this too in this PR? 

Get Subscribed SKUs

Reply:
No I did not.